src/services/infra/InfraService.ts

```diff
@@ -96,6 +116,7 @@ export default class InfraService {
 
       // 5. Set up permissions
       await this.reposService.setRepoAndTeamPermissions(repoName)
+      await this.collaboratorsService.create(repoName, memberEmail, true)
```

Comment:
Hmm just to clarify, why is acknowledged = true over here ah? In what case will acknowledged = false ah?

Reply:
We're reusing the collaborators method here - this was a verification that when adding collaborators (non govt email users), that the user had acknowledged on the frontend. It's not applicable here, so it's always true